AsyncDisplayKit/Details/_ASDisplayLayer.h

```diff
@@ -70,6 +70,20 @@ typedef BOOL(^asdisplaynode_iscancelled_block_t)(void);
 @end
 
 /**
+ * Optional methods that the view associated with an _ASDisplayLayer can implement. 
+ * This is distinguished from _ASDisplayLayerDelegate in that it points to the _view_
+ * not the node. Unfortunately this is required by ASCollectionView, since we currently
+ * can't guarantee that an ASCollectionNode exists for it.
+ */
+@protocol ASCALayerExtendedDelegate
```

Comment:
I think `ASCALayerExtendedDelegate` is more appropriate because we're literally piggy-backing on top of CALayerDelegate here, and I want to distinguish it from `_ASDisplayLayerDelegate`.